fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -280,13 +328,31 @@ case "$BiasCorrection" in
 		;;
 esac
 
-
 ########################################## DO WORK ########################################## 
 
 T1wFolder="$Path"/"$Subject"/"$T1wFolder"
 AtlasSpaceFolder="$Path"/"$Subject"/"$AtlasSpaceFolder"
 ResultsFolder="$AtlasSpaceFolder"/"$ResultsFolder"/"$NameOffMRI"
 
+# set reference and check if external reference (if one is specified) exists 
+
+if [ ! $fMRIReference = "NONE" ] ; then  
+  fMRIReferencePath="$Path"/"$Subject"/"$fMRIReference"
+  log_Msg "Using reference image from ${fMRIReferencePath}"
+  fMRIReferenceImage="$fMRIReferencePath"/"$ScoutName"_gdc
+
+  if [ "$fMRIReferencePath" = "$fMRIFolder" ] ; then
+    log_Err_Abort "Specified BOLD reference (--fmriref) is the same as the current BOLD (--fmriname)!"
+  fi
+
+  if [ `${FSLDIR}/bin/imtest ${fMRIReferenceImage}` -eq 0 ] ; then
+    log_Err_Abort "Specified BOLD Reference ${fMRIReferenceImage} does not exist!"
```

Comment:
`log_Err_Abort "Intended BOLD Reference does not exist (${fMRIReferenceImage})!"`

Reply:
ok